specs/sb/api.md

```diff
@@ -342,6 +400,45 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
         }, {
           "content": "40 concurrent connections"
         }]
+      },
+      "schemas": {
+        "service_instance": {
+          "parameters": {
+            "required": [
+              "fake_required_property"
+            ],
+            "properties": {
+              "fake_required_property": {
+                "type": "string"
+              }
+            }
+          },
+          "outputs": {
+            "properties": {
```

Comment:
Your example value of `outputs` is `dashboard_url`, which is an optional top-level field. `schemas` is effectively an opaque object specific to a broker. Are you proposing that `dashboard_url` is no longer part of the API spec? 

IMO, `outputs` is beyond the scope of the discussed use case. We're not trying to schemify all the things. The scope is very specific: expose what fields a client could send in the `parameters` field on create, update, and bind. This `outputs` field defines a broker's response for those operations. Arguably this is not necessary, because the broker's response is already defined by the API spec.